kani-compiler/src/kani_compiler.rs

```diff
@@ -25,6 +25,7 @@ use crate::kani_middle::reachability::filter_crate_items;
 use crate::kani_middle::stubbing::{self, harness_stub_map};
 use crate::kani_queries::QueryDb;
 use crate::session::init_session;
+use cbmc::{InternString, InternedString};
```

Comment:
Why is the `InternString` import needed?

Reply:
Because I'm using the interned string id as the hash now for the harnesses. I could just use the harness names directly, but we usually store the intern version in our data structures.